script.js

```diff
@@ -0,0 +1,298 @@
+localStorage.setItem('caps', 'smallLetter');
+localStorage.setItem('lang', 'Eng');
+let lang = localStorage.getItem('lang');
+if (sessionStorage.hasOwnProperty('lang')) {
+    lang = sessionStorage.getItem('lang');
+}
+const body = document.querySelector('body');
+const div = document.createElement('div');
```

Comment:
It would be better if the variable `div` was more specific. For example, `keyboardContainer`.